pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1293,6 +1299,28 @@ func setJvmDefault(jvm *api.JvmObject) {
 	}
 }
 
+func setKafkaExtDefault(kafka *api.KafkaExt) {
+	if kafka != nil {
+
+		if kafka.BootstrapServers == "" {
```

Comment:
for strings it is better to use len(xxx) == 0 or < 0..

Reply:
ok

Reply:
ok, but  == "" or != "" seems used https://github.com/kiegroup/kie-cloud-operator/blob/main/pkg/controller/kieapp/defaults/defaults.go#L966